lib/db/schema/patch-067-068.sql

```diff
@@ -0,0 +1,155 @@
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `shortCodeHash` BINARY(32) DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE INDEX `unverifiedTokens_shortCodeHash`
+ON `unverifiedTokens` (`shortCodeHash`)
+ALGORITHM = INPLACE LOCK=NONE;
+
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `shortCodeExpiresAt` BIGINT UNSIGNED DEFAULT NULL,
```

Comment:
You should be able to add both of these columns in a single `ALTER TABLE` statement, which will be easier for ops to run.